helm/dicom-cast/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: dicom-cast
+description: A Helm chart for DICOM Cast
```

Comment:
Chart description should stand on its own and contain a link to the project  (ie if it was not being nestled inside of this repository)